src/improvements/justfile

```diff
@@ -171,16 +171,97 @@ simulate-all-templates:
         exit 1
     fi
 
-simulate-verify-task TASK="":
+# Simulate a task locally and against Tenderly.
+# TASK: The fullpath to the task to simulate.
+# SAFE_NAME: The name of the safe to simulate the task for (council, foundation). Leave blank for non-nested tasks.
+#
+# Example:
+# cd src/improvements
+# just simulate-tenderly tasks/sep/000-opcm-upgrade-v200 council
+# just simulate-tenderly tasks/sep/003-unichain-superchain-config-fix
+simulate-tenderly TASK="" SAFE_NAME="":
+    #!/usr/bin/env bash
+    set -euo pipefail
+    root_dir=$(git rev-parse --show-toplevel)
+    forge build
+
+    # If the task path doesn't start with a slash, append the current directory to the task path
+    task_path="{{TASK}}"
+    if [[ "$task_path" != /* ]]; then
+        task_path="${PWD}/${task_path}"
+    fi
+
+    ${root_dir}/src/improvements/script/simulate-tenderly.sh "$task_path" {{SAFE_NAME}}
+
+# Simulate all non-terminal tasks locally and against Tenderly, then verify the domain separator and message hash match against the values in the VALIDATION.md file.
+#
+# Parses these strings from the VALIDATION.md file:
+# ### Security Council / Optimism Foundation / Chain Governor / Child Safe 1 / Child Safe 2 / Single Safe Signer Data
+# Domain Hash: `<hash>`
+# Message Hash: `<hash>`
+verify-non-terminal-tasks:
     #!/usr/bin/env bash
     set -euo pipefail
     root_dir=$(git rev-parse --show-toplevel)
 
     forge build
 
-    # If the task is nested then we only simulate as the foundation.
-    # In the future we could simulate as other nested safes. 
-    # For testing purposes, we do not gain anything by simulating as other nested safes.
-    nested_safe_name="foundation"
+    for task in ${root_dir}/src/improvements/tasks/*/*; do
+        if [ -d "$task" ]; then
 
-    ${root_dir}/src/improvements/script/simulate-verify-task.sh {{TASK}} $nested_safe_name
+            # Read the README.md file and skip if it contains "Status: [SIGNED] or "Status: [EXECUTED]"
```

Comment:
![CleanShot 2025-03-27 at 16 35 17@2x](https://github.com/user-attachments/assets/8c46ad37-2c19-4e89-9aee-f45767dd2c9d)
Same thing with this comment.

Reply:
Added the missing quotes, but I think that you are just using the wrong highlighting.